bytes/concat.ts

```diff
@@ -3,6 +3,7 @@
 
 /** Concatenate the given arrays into a new Uint8Array.
  *
+ * @example
  * ```ts
  * import { concat } from "https://deno.land/std@$STD_VERSION/bytes/concat.ts";
  * const a = new Uint8Array([0, 1, 2]);
```

Comment:
unrelated, but i just noticed that the codeblock doesnt have closing backticks. could you add them?